examples/pydantic/pydantic_with_pre_planned_tool.py

```
@@ -61,26 +64,50 @@ def implementation(function_parameters: str) -> str:
+    Panel.fit(
+        Markdown(result.output),
+        title="🤖 Agent Result",
+        border_style="blue",
+        padding=(1, 2),
     )
 )
 
-result = agent.run_sync("tell me about github user zixuan-x")
-print(result)
+# Also display some metadata about the result
+console.print(f"\n[bold green]✅ Task completed successfully![/bold green]")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)